controllers/storagecluster/volumegroupsnapshotterclasses.go

```diff
@@ -111,6 +112,12 @@ func (obj *ocsGroupSnapshotClass) ensureCreated(r *StorageClusterReconciler, ins
 		cephFsGroupSnapshotClass,
 	}
 
+	if instance.Spec.ExternalStorage.Enable {
```

Comment:
Why is there a need for a check. I believe that this code only runs in external mode